android/src/main/java/info/whitebyte/hotspotmanager/ClientScanResult.java

```diff
@@ -17,48 +17,48 @@
 package info.whitebyte.hotspotmanager;
 
 public class ClientScanResult {
-  private String IpAddr;
```

Comment:
You may wanna revert the formatting. Don't think its necessary to make this change.